Server/Terrain.py

```diff
@@ -0,0 +1,91 @@
+import random as ra
+import math as mat
+import bresenham as br
+
+
+SCREEN_WIDTH = 800
+SCREEN_HEIGHT = 600
```

Comment:
Is it related to Terrain? I think other modules/classes have to use screen size too! Maybe it's good here, just think about it.

Reply:
It is not, once the need of other modules within Server dir. for these will arise I will come up with a common, more elegant solution.